### PR TITLE
Incorporate SignalrClient with AnyConnect Platform libraries

### PR DESCRIPTION
**Pull request by seym45**
- Added build script for Android/RPi/Ubuntu
- Added necessary flags to CMake files

### CODE REVIEW

File: build.RPi.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+cd ${SIGNALRCLIENT_BUILD_DIR}
+rm -rf *
+
+
+PLATFORM_ROOT=$SIGNALR_ROOT/.build_RPi
+BOOST_ROOT=${PLATFORM_ROOT}/boost-1.61-RPi
+CPPRESDSDK_ROOT=${PLATFORM_ROOT}/accessapi-artifacts-RPi-ms/cpprest
+OPENSSL_ROOT=${PLATFORM_ROOT}/openssl-1.0.2l-RPi
+
+command="cmake .. -DCMAKE_TOOLCHAIN_FILE=$HOME/projects/.launchers/automation/build/RPi/RPi.toolchain  \
```
Comment (KhaledMahmood) [inline build.RPi.sh]: Please use the path "/opt/RPi/RPi.toolchain" instread of "$HOME/projects/.launchers/automation/build/RPi/RPi.toolchain". I guess /opt/ is the actual directory for all toolchains.

**Review by KhaledMahmood — CHANGES_REQUESTED** [review]
Please consider changing the toolchain path for RPi

**Review by KhaledMahmood — APPROVED** [review]
Looks good. Approved.

File: build.RPi.sh (hunk 1 above)
Comment (seym45) [reply build.RPi.sh]: Changed accordingly.

**Review by KhaledMahmood — APPROVED** [review]
Looks good.

**Review by KhaledMahmood — APPROVED** [review]
Looks good.

**Pull request merged**